### ISSUE TITLE
Load more feature

### ISSUE THREAD

**Issue by nguyenhoanglam** [bignerdranch/expandable-recycler-view#130]
ExpandableRecyclerView will be great if it has an load more listener callback for each child list. I'm waiting for this feature for a long time. Hope the next version will include this feature.

**Comment by dgreenhalgh** [comment]
Hey Nguyen - could you clarify? My understanding is that you're wanting to implement a partially expanded list of children. Is this correct?

**Comment by nguyenhoanglam** [comment]
@dgreenhalgh No. What I want is when I scroll the child list to the end, I need a callback to let me know that event. So I will load more data and add to the child list. This called load more in RecyclerView.

**Comment by paul-turner** [comment]
@nguyenhoanglam are there existing implementations for this with a normal recycler view?
This might be something better suited for the layout manager than a custom adapter.
